tests/integration/conftest.py

```diff
@@ -309,6 +309,21 @@ def vmc_nsx_connect(vmc_config):
     )
 
 
+@pytest.fixture(scope="session")
+def vmc_connect(vmc_config):
+    vmc_config = vmc_config["vmc_connect"]
```

Comment:
**question** why isn't this just `return vmc_config["vmc_connect"]`? At a glance it looks like that would remove the need for two of these fixtures.

Reply:
updated